spec/views/categories/index_spec.rb

```diff
@@ -28,4 +28,4 @@
   #     end
   #     expect(current_path).to match group_expenses_path(@category)
   # end
-end
\ No newline at end of file
+end
```

Comment:
![Screenshot from 2023-03-01 09-30-22](https://user-images.githubusercontent.com/83042742/222073016-65314be5-e3b0-44a3-8617-d07a706b73bf.png)
- [ ] `spec/views/transactions/index_spec.rb` file: For clean code and best practices, kindly ensure that you remove all the commented and used code lines from your code base